appservice/src/deploy/localGitDeploy.ts

```diff
@@ -58,3 +59,23 @@ export async function localGitDeploy(client: SiteClient, fsPath: string): Promis
     ext.outputChannel.appendLine(formatDeployLog(client, (localize('localGitDeploy', `Deploying Local Git repository to "${client.fullName}"...`))));
     await waitForDeploymentToComplete(client, kuduClient);
 }
+
+enum RunFromPackageAliases {
+    fromPackage = 'WEBSITE_RUN_FROM_PACKAGE',
+    fromZip = 'WEBSITE_RUN_FROM_ZIP'
+}
+
+async function verifyNoRunFromPackageSetting(client: SiteClient): Promise<void> {
+    let updateSettings: boolean = false;
+    const applicationSettings: StringDictionary = await client.listApplicationSettings();
+    for (const key of Object.keys(RunFromPackageAliases)) {
+        const runFromPackageSettingName: RunFromPackageAliases = <RunFromPackageAliases>RunFromPackageAliases[key];
+        if (applicationSettings.properties && applicationSettings.properties[runFromPackageSettingName]) {
+            delete applicationSettings.properties[runFromPackageSettingName];
+            updateSettings = true;
```

Comment:
nit: Add:
```
ext.outputChannel.appendLine(formatDeployLog(client, ...
```
with a message that you're removing the setting

Reply:
It's not worth promping the user, right?